examples/webextension/background.js

```diff
@@ -0,0 +1,15 @@
+const TRACKING_ID = 'UA-XXXXXXXX-YY';
+
+const { sendEvent } = new Metrics({
```

Comment:
Great question. WebExtensions background scripts seem to just get loaded into the auto-generated background page, script tag style (implicit global binding). Trying to require() the file threw an error. ES6 modules might work, but I didn't try that: I didn't want to lock experiment authors into dealing with ES6 in their build scripts.